lib/pages/login/create_profile.dart

```diff
@@ -64,9 +67,14 @@ class _CreateAccountPageState extends ConsumerState<CreateAccountPage> {
       _isloading = false;
       return;
     }
+
+    Uint8List? imageBytes = _image == null ? null : await _image!.readAsBytes();
```

Comment:
Let's Wrap everything with `try` `catch` block. Executing `async` functions like this is dangerous

Reply:
Will do 💯